analytics_data_api/v0/tests/test_connections.py

```diff
@@ -33,7 +33,7 @@ def test_signing(self):
         self.assertTrue('my_access_key' in auth_header)
 
     def test_timeout(self):
-        def fake_connection(_address):
+        def fake_connection(_address, _timeout):
```

Comment:
Yes; the upgraded Boto library passes two arguments here, so we get a TypeError when testing otherwise.